forms-flow-web/src/modules/userDetailReducer.js

```diff
@@ -1,17 +1,37 @@
 import ACTION_CONSTANTS from "../actions/actionConstants";
-import { setShowApplications, setShowViewSubmissions } from "../helper/user";
+import {
+  setShowApplications,
+  setShowViewSubmissions,
+  setUserRolesToObject,
+} from "../helper/user";
 import { LANGUAGE } from "../constants/constants";
+import { setFormAndSubmissionAccess } from "../helper/access";
 const getLanguages = localStorage.getItem("languages");
 const initialState = {
   bearerToken: "",
   roles: "",
+  roleIds: localStorage.getItem("roleIds")
+    ? setUserRolesToObject(JSON.parse(localStorage.getItem("roleIds")))
+    : {},
+  formAccess: localStorage.getItem("roleIds")
```

Comment:
FormAccess and submission Access needed for designer case only

Reply:
if (ROLE.DESIGNER === undefined) {
    return [];
  }